### DISCUSSION TITLE
Review the logs from running command

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#116] [Q&A — Plugin]
Hi,

Is there any way to store and review the notifications from running command, like which files were changed, published, error?

Currently, I have to open web browser to check the github repo, to be sure that certain files were uploaded to repo

I also acknowledge that there is an option to display all notifications in Obsidian at the runtime of plugin, but the notifications disappear too fast, can't read in details.

**Comment by Mara-Li** [comment]
You can open the console with CTRL+MAJ+I but I can make a new log with a list of changed file!

**Comment by Mara-Li** [comment]
Hi! The new beta add the option to see the impact on the repo when sharing file :)
